make.js

```diff
@@ -345,8 +345,8 @@ target.testLegacy = function() {
     run('mocha ' + testsSpecPath, /*inheritStreams:*/true);
 }
 
-target.package = function() {
-    // validate powershell 5
+target.layout = function() {
```

Comment:
nit: rather than introduce another make-target, it might keep things a little simpler to just log and skip the nuget packaging step if --version is not specified. today it errors out.

note, the "publish" target does more than just publish the nuget package, it also stores the layout.